### PR TITLE
Handle cases of count(null) in Container.php

### PR DESCRIPTION
**Pull request by FelixAuer**
Sometimes count() gets called on null, which is not possible in PHP7.2.

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
Can you up the various dependencies to reflect the PHP 7.2 min version? https://www.php.net/manual/en/function.is-countable.php

File: Config/Container.php [hunk 1]
```diff
@@ -694,15 +694,15 @@ function toArray($useAttr = true)
         $array[$this->name] = array();
         switch ($this->type) {
             case 'directive':
-                if ($useAttr && count($this->attributes) > 0) {
+                if ($useAttr && is_countable($this->attributes) && count($this->attributes) > 0) {
```
Comment (devnix) [inline Config/Container.php]: Would change it to a `if ($useAttr && !empty($this->attributes)) {` maybe?

File: Config/Container.php [hunk 2]
```diff
@@ -694,15 +694,15 @@ function toArray($useAttr = true)
...
                     $array[$this->name]['#'] = $this->content;
                     $array[$this->name]['@'] = $this->attributes;
                 } else {
                     $array[$this->name] = $this->content;
                 }
                 break;
             case 'section':
-                if ($useAttr && count($this->attributes) > 0) {
+                if ($useAttr && is_countable($this->attributes) && count($this->attributes) > 0) {
```
Comment (devnix) [inline Config/Container.php]: Would change it to a `if ($useAttr && !empty($this->attributes)) {` maybe?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Fork of pear/Config for PHP7.2
+
+This fork handles cases when count() is called on null.
```
Comment (devnix) [inline README.md]: I guess this should be gone

**Comment by FelixAuer** [comment]
@CloCkWeRX  I've added the min version to composer.json and package.xml. Are there any other places to add it?

File: README.md (hunk 3 above)
Comment (FelixAuer) [reply README.md]: Agree, it's gone now.

File: Config/Container.php (hunk 1 above)
Comment (FelixAuer) [reply Config/Container.php]: Good call

File: Config/Container.php (hunk 2 above)
Comment (FelixAuer) [reply Config/Container.php]: Thank you, much cleaner this way.

**Review by CloCkWeRX — APPROVED** [review]

**Pull request merged**